src/Database/Type/GeometryType.php

```diff
@@ -23,6 +23,13 @@ class GeometryType implements TypeInterface
      */
     protected string $name = 'geometry';
 
+    /**
+     * Decorator method.
+     *
+     * @var (callable(\Brick\Geo\Geometry $geometry): \Brick\Geo\Geometry)|null
+     */
+    protected $decorator = null;
```

Comment:
Depending on the PHP version we're supporting this might be possible:

```suggestion
    protected ?callable $decorator = null;
```